pom.xml

```diff
@@ -39,6 +39,8 @@
     <sonar.c.file.suffixes>-</sonar.c.file.suffixes>
     <sonar.cpp.file.suffixes>-</sonar.cpp.file.suffixes>
     <sonar.objc.file.suffixes>-</sonar.objc.file.suffixes>
+    <skipAPIDescription>true</skipAPIDescription>
```

Comment:
After thinking about this again and checking the current artifacts in the I-build repo, I think we must not disable the api-description generation. Skipping the analysis is fine, but the description is currently generated and the o.e.swt Plugin as well as its platform specific fragments (I only checked windows) contain an `.api_description` file.

I think they have to be retained, so that part should be reverted and the api-description generation has to be fixed in another way.

Reply:
The api-description works fine, so we can simply set this to false here, or delete the line ...

Reply:
Then better remove it or one day someone will wonder what this is about and will spend some time to find out that is actually unused.

Reply:
Done in #598.